CHANGELOG.md

```diff
@@ -29,3 +29,15 @@
 [0.60.1]: https://github.com/marshmallow-insurance/smores-react/compare/v0.60.0...v0.60.1
 [0.60.0]: https://github.com/marshmallow-insurance/smores-react/compare/v0.59.0...v0.60.0
 [0.59.0]: https://github.com/marshmallow-insurance/smores-react/compare/v0.58.0...v0.59.0
+
+## [0.61.0] - 2021-04-21
+### Changed
+- tertiary button styling
+
+### Added
+- bg4 colour to theme
+
+## [0.65.0] - 2021-05-04
+### Added
+- New Chip component
+
```

Comment:
Can you move these at the top to maintain the order of the changelog?
Latest updates should be added at the top of the file.

Also can you add the new release link above 0.63.0? This is needed to properly link the changelog file to the github changes page. Should looks like:
`[0.65.0]: https://github.com/marshmallow-insurance/smores-react/compare/v0.64.0...v0.65.0`

Reply:
No problem! Thanks for the instructions

Reply:
Do I add the new release link after my PR is merged? As I haven't created v0.65.0 yet, right?